node-webservice/react-client/src/Components/Grid.jsx

```diff
@@ -21,6 +24,39 @@ class Grid extends Component {
 
         return partsArray;
     }
+    
+    typeChooser(){
+        return <table>
+            <tr>
+                <input  type="radio"  value="block"  id="block" class="input-hidden"  name="controller"
+                defaultChecked = "checked"  
+                onClick={() => this.props.onChangeType("block") } style={ this.style(Block) }
+                />
+                <label for="block">  
+                    <img style={ this.style(Block) } src={Block} />
```

Comment:
The img tags should have an alt property https://www.w3schools.com/tags/att_img_alt.asp